src/release.config.mjs

```diff
@@ -21,11 +24,16 @@ export default {
     [
       '@semantic-release/exec',
       {
+        analyzeCommitsCmd: 'echo "minor"',
+        generateNotesCmd: 'echo "chore: bump version"',
```

Comment:
you can also move this to `plugins.RELEASE`.

Reply:
plugins.RELEASE:

[
 { '@semantic-release/exec', { .... } },
 { your plugin }
]

Reply:
got it